package.json

```diff
@@ -28,7 +28,7 @@
     "immutable": "~3.8.1",
     "jquery": "^3.2.1",
     "lodash": "~4.17.4",
-    "moment": "~2.29.4",
+    "moment": "^2.29.4",
```

Comment:
just curious, why did this change?

Reply:
I think this happens because I reinstalled this lib. But We can leave it the way it was.

Reply:
if this is unrelated to this work I think we should leave the way it was.